docusaurus/docs/extensions/publishing.md

```diff
@@ -9,7 +9,58 @@ As discussed in the [Getting Started](./extensions-getting-started#creating-a-re
 
 > Note: An explanation on the workflow files can be found in the [Additional Release Configuration](#additional-release-configuration) section.
 
-# Publish Script Usage
+## Automatic Approach - Triggering a Github Workflow on Tagged Release
+
+If your extensions repository doesn't include a github workflow to automate the publishing procedure (igore this step if you have it already), you can create one in `.github/workflows/build.yaml` on your extension folder, with the following content:
+
+```workflow
+name: Build and Release Extension Charts
+
+on:
+  workflow_dispatch:
+  release:
+    types: [released]
+
+defaults:
+  run:
+    shell: bash
+    working-directory: ./
+
+jobs:
+  build-extension-charts:
+    uses: rancher/dashboard/.github/workflows/build-extension-charts.yml@master
+    permissions:
+      actions: write
+      contents: write
+      deployments: write
+      pages: write
+    with:
+      target_branch: gh-pages
+
+```
+
+This will leverage the usage of our reusable workflow and give your Github extension repo the ability to publish to the `gh-pages` branch via a **Tagged Release**. 
+
+It's mandatory that release type is **Tagged Release**, otherwise the `build-extension-charts.yml` workflow won't run properly, which is crucial for the success of the publish operation
```

Comment:
```suggestion
It's mandatory that release type is **Tagged Release**, otherwise the `build-extension-charts.yml` workflow won't run properly, which is crucial for the success of the publish operation.
```